include/EASTL/internal/hashtable.h

```diff
@@ -1263,10 +1263,15 @@ namespace eastl
 		node_type* DoAllocateNode(value_type&& value);
 		node_type* DoAllocateNode(const value_type& value);
 
-		eastl::pair<iterator, bool> DoInsertKey(true_type, const key_type& key);
-		iterator                    DoInsertKey(false_type, const key_type& key);
-		eastl::pair<iterator, bool> DoInsertKey(true_type, key_type&& key);
-		iterator                    DoInsertKey(false_type, key_type&& key);
+		eastl::pair<iterator, bool> DoInsertKey(true_type, const key_type& key, hash_code_t c);
```

Comment:
Add a comment that describes the purpose of these overloads.

Reply:
just to clarify - comment in code, right?

added overloads mimics current(old) API, so they kept just for compatibility reasons (if someone use it already).

Reply:
Yes.  Comments in code as hints to users who don't know why both sets of API's exist and why they should reach for one or the other.

Reply:
done